### PR TITLE
Simplify Model Behavior

### PR DESCRIPTION
**Pull request by Coffee2Bits**
- Removed the required behavior field.
- Added an external model for the User

### CODE REVIEW

File: model/flow/External.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+model:
+  name: User
+  behavior:
```
Comment (Coffee2Bits) [inline model/flow/External.yaml:3]: This is technically valid, but it looks weird to define an empty behavior for a model with no inputs/outputs.

Comment (cvchaparro) [reply model/flow/External.yaml:3]: I think what we'd actually want to do is define it like this (i.e. an empty list of behaviors):

```yaml
model:
  name: User
  behavior: []
```

Comment (cvchaparro) [reply model/flow/External.yaml:3]:
```suggestion
  behavior: []
```

**Comment by cvchaparro** [comment]
Other than my previous comment, it looks good. My only question is regarding the validator tests. As they were, the tests all assumed behaviors required acceptance. This is no longer a valid assumption with this change so, I'd think we'd need to change those things.

For example:
- [Here](https://github.com/jondavid-black/AaC/blob/main/tests/test_validator.py#L83-L87) it might be good to add a behavior without an acceptance and make sure validation doesn't fail on that.

**Review by cvchaparro — CHANGES_REQUESTED** [review]
Other than my previous comment, it looks good. My only question is regarding the validator tests. As they were, the tests all assumed behaviors required acceptance. This is no longer a valid assumption with this change so, I'd think we'd need to change those things.

For example:
- [Here](https://github.com/jondavid-black/AaC/blob/main/tests/test_validator.py#L83-L87) it might be good to add a behavior without an acceptance and make sure validation doesn't fail on that.

File: model/flow/External.yaml (hunk 1 above)
Comment (jondavid-black) [reply model/flow/External.yaml:3]: Why is it valid?  Behavior has required fields of name and type.  Given these fields aren't present shouldn't it be invalid?

**Review by cvchaparro — APPROVED** [review]

**Pull request merged**